model_authorization_info_info.go

```diff
@@ -0,0 +1,144 @@
+/*
+OCPI tokens module
+
+Specification for OCPIs tokens handlers
+
+API version: 2.2.1
+*/
+
+// Code generated by OpenAPI Generator (https://openapi-generator.tech); DO NOT EDIT.
+
+package OCPI
+
+import (
+	"encoding/json"
+)
+
+// checks if the AuthorizationInfoInfo type satisfies the MappedNullable interface at compile time
+var _ MappedNullable = &AuthorizationInfoInfo{}
+
+// AuthorizationInfoInfo struct for AuthorizationInfoInfo
+type AuthorizationInfoInfo struct {
+	Language string `json:"language"`
+	Text string `json:"text"`
+}
+
+// NewAuthorizationInfoInfo instantiates a new AuthorizationInfoInfo object
+// This constructor will assign default values to properties that have it defined,
+// and makes sure properties required by API are set, but the set of arguments
+// will change when the set of required properties is changed
+func NewAuthorizationInfoInfo(language string, text string) *AuthorizationInfoInfo {
```

Comment:
InfoInfo is not a good naming convention.

Reply:
resolved ✅